api/dispatcher.proto

```diff
@@ -48,6 +49,24 @@ service Dispatcher { // maybe dispatch, al likes this
 	rpc Tasks(TasksRequest) returns (stream TasksMessage) {
 		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-worker" roles: "swarm-manager" };
 	};
+
+	// CreateContainerAttachment allows the node to request the resources
+	// allocation needed for a container attachment on the specific node.
+	rpc CreateContainerAttachment(CreateContainerAttachmentRequest) returns (CreateContainerAttachmentResponse) {
```

Comment:
After this comment, I moved the creation/removal methods to the controlapi, but then the security concern arose, given we would need to open the controlapi to the workers.

So it was discussed offline to go ahead with those methods in the dispatcher API until we find a better place for them.